snapd-qt/Snapd/qt-visibility.h

```diff
@@ -0,0 +1,5 @@
+#if defined(LIBSNAPDQT)
```

Comment:
`#include <QtCore/QtGlobal>` for `Q_DECL_IMPORT`

Reply:
Done. Thanks!